convert2rhel/grub.py

```diff
@@ -172,7 +172,7 @@ def get_device_number(device):
         raise BootloaderError("Unable to get information about the '%s' device" % device)
     # We are spliting the partition entry number, and we are just taking that
     # output as our desired partition number
-    if not output:
+    if not output or output == "\n":
```

Comment:
Can't we just strip the output? The check should work after that

Reply:
Agreed, `output.strip()` should be used instead. Preferably done just after the `run_subprocess()` call

```
>>> '\n'.strip()
''
```